packages/debian/distributions

```diff
@@ -0,0 +1,7 @@
+Origin: Scality
+Label: metalk8s
+Suite: stable
+Codename: bionic
+Architectures: amd64
+Components: main metalk8s-bionic metalk8s-bionic-backports metalk8s-bionic-updates metalk8s-kubernetes-xenial metalk8s-salt_ubuntu1804 metalk8s-scality
+Description: Build repository for Debian
```

Comment:
Not sure this description is accurate. It should describe the repository, so let's go for something like:

```
Description: MetalK8s repository for bionic
```

Reply:
Indeed, it's from my tests, I will change it.